Scripts/Modules/ObjectPlacementModule.cs

```diff
@@ -11,8 +11,8 @@ internal class ObjectPlacementModule : MonoBehaviour, IUsesSpatialHash
 		const float kInstantiateFOVDifference = -5f;
 		const float kGrowDuration = 0.5f;
 
-		public Action<GameObject> addToSpatialHash { get; set; }
-		public Action<GameObject> removeFromSpatialHash { get; set; }
+		public Action<GameObject> addToSpatialHash { private get; set; }
```

Comment:
That would require explicit interface declaration and access, no? e.g. IUsesSpatialHash.addToSpatialHash { ... }